hedera-node/hapi-fees/src/main/java/com/hedera/node/app/hapi/fees/usage/crypto/CryptoOpsUsage.java

```diff
@@ -86,6 +87,9 @@ public void cryptoTransferUsage(
         incRb += TOKEN_ENTITY_SIZES.bytesUsedToRecordTokenTransfers(
                 weightedTokensInvolved, weightedTokenXfers, xferMeta.getNumNftOwnershipChanges());
         accumulator.addRbs(incRb * USAGE_PROPERTIES.legacyReceiptStorageSecs());
+
+        // Add rbs for auto associations
+        accumulator.addRbs((xferMeta.getNumFungibleTokenTransfers() + xferMeta.getNumNftOwnershipChanges()) * THREE_MONTHS_IN_SECONDS * CREATE_SLOT_MULTIPLIER);
```

Comment:
I added some default values here as we don't have the actual number of new associations in the current transfer.